draft-ietf-quic-transport.md

```diff
@@ -2352,171 +2361,136 @@ shown in the following figure and described below.
+                            |        |
+                            | closed |
+                            |        |
+                            +--------+
+~~~
+{: #stream-state-send title="Stream States for Sending"}
 
-   send:   endpoint sends this frame
-   recv:   endpoint receives this frame
 
-   STREAM: a STREAM frame
-   FIN:    FIN flag in a STREAM frame
-   RST:    RST_STREAM frame
-   MSD:    MAX_STREAM_DATA frame
-   SB:     STREAM_BLOCKED frame
-~~~
-{: #stream-lifecycle title="Lifecycle of a stream"}
+### Stream States for Receiving
 
-Note that this diagram shows stream state transitions and the frames and flags
-that affect those transitions only.  It is possible for a single frame to cause
-two transitions: receiving a RST_STREAM frame, or a STREAM frame with the FIN
-flag cause the stream state to move from "idle" to "open" and then immediately
-to one of the "half-closed" states.
+{{stream-state-recv}} illustrates the states and transitions that apply to a
+stream at the receiving endpoint.
 
-The recipient of a frame that changes stream state will have a delayed view of
-the state of a stream while the frame is in transit.  Endpoints do not
-coordinate the creation of streams; they are created unilaterally by either
-endpoint.  Endpoints can use acknowledgments to understand the peer's subjective
-view of stream state at any given time.
+~~~
+                            +--------+
+                            |        |
+                            |  idle  |
+                            |        |
+                            +--------+
+                                 |
+                     receive STREAM, RST_STREAM or
+                           STREAM_BLOCKED
+                     or receive these frames for a
+                         higher-numbered stream
+                                 |
+                                 v
+                            +--------+
+                            |        |
+                            |  open  |
+                            |        |
+                            +--------+
+                                 |
+                     receive STREAM with FIN flag
+                        receive RST_STREAM
+                                 |
+                                 v
+                            +--------+
+                            |        |
+                            | closed |
+                            |        |
+                            +--------+
+~~~
+{: #stream-state-recv title="Stream States for Receipt"}
 
-In the absence of more specific guidance elsewhere in this document,
-implementations SHOULD treat the receipt of a frame that is not expressly
-permitted in the description of a state as a connection error (see
-{{error-handling}}).
+The recipient of a stream will have a delayed view of the state of streams at
+its peer.  Loss or delay of frames can cause frames to arrive out of order.
 
 
-### idle
+### The "idle" State
```

Comment:
It means "I won't send," I'm fairly sure.  That's the allusion to #171, which defines a separate "I won't receive any more" frame.